src/modules/test/tests/mim/sample.json

```diff
@@ -0,0 +1,124 @@
+{
+    "name": "SampleCommandRunnerModel",
+    "type": "MimModel",
+    "contents": [
+      {
+        "name": "CommandRunner",
+        "type": "mimComponent",
+        "contents": [
+          {
+            "name": "commandStatus",
+            "type": "mimObject",
+            "desired": false,
+            "schema": {
+              "fields": [
+                {
+                  "name": "commandId",
+                  "schema": "string"
```

Comment:
Why is this duplicating mim/CommandRunner.json?

Can you use existing MIMs instead of made up samples which aren't even so made up after all?

Reply:
i will pull this out actually, the final revision will make use of what we have in our modules directory